### PR TITLE
data:GarimaSaroj

### PR DESCRIPTION
**Pull request by AndroGari**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6091"></a>

### CODE REVIEW

**Review by shyamtawli — CHANGES_REQUESTED** [review]
filename should be your github username like `AndroGari.json`

File: GarimaSaroj.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Garima Saroj",
+  "type": "personal",
+  "bio": "artist turned **AI aficionado**. MLOps enthusiast and a designer",
+  "links": [
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/garimasaroj/",
+      "icon": "FaLinkedin"
+    }
```
Comment (shyamtawli) [inline GarimaSaroj.json:10]:
```suggestion
    },
```

File: GarimaSaroj.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Andro_Gari",
+      "icon": "FaTwitter"
+    }
```
Comment (shyamtawli) [inline GarimaSaroj.json:15]:
```suggestion
    },
```

File: GarimaSaroj.json (hunk 1 above)
Comment (imbeshat) [inline GarimaSaroj.json:10]:
```suggestion
    },
```

File: GarimaSaroj.json (hunk 2 above)
Comment (imbeshat) [inline GarimaSaroj.json:15]:
```suggestion
    },
```

File: GarimaSaroj.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/AndroGari",
+      "icon": "FaGithub"
+    }
```
Comment (imbeshat) [inline GarimaSaroj.json:20]:
```suggestion
    },
```

Comment (shyamtawli) [inline GarimaSaroj.json:20]:
```suggestion
    },
```

**Review by shyamtawli — CHANGES_REQUESTED** [review]
add `,` after every completed object

**Review by imbeshat — CHANGES_REQUESTED** [review]
Also, move your file to ```data``` folder.

**Review by iamravi1 — CHANGES_REQUESTED** [review]
JSON file name should be your username and remaining looks good

**Comment by SaraJaoude** [comment]
> JSON file name should be your username and remaining looks good

@iamravi1  when reviewing, please look at the conversation above before making a comment. This has already been mentioned and adding the same comment again can create confusion for the author.

**Comment by iamravi1** [comment]
Ok sure

**Review by md-abid-hussain — APPROVED** [review]
LGTM

**Review by iamravi1 — APPROVED** [review]
LGTM 🎉

**Comment by SaraJaoude** [comment]
@iamravi1 and @md-abid-hussain 

your comments of LGTM cause confusion to the author and also maintainers because this profile is not ready to be merged - given than the JSON file is not in the correct directory and the file name needs to match the username. Before adding LGTM on a Pull Request, please double check this as much as possible. 


@AndroGari please let us know if you need help with the changes.

**Comment by amandamartin-dev** [comment]
Hello @AndroGari it has been a few weeks please let us know if you are able to address the changes or if you are having any trouble understanding what is needed to get your profile merged! We are happy to help.  If we do not hear a response, we will close this PR for now but you are welcome to resubmit at any time.

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on LinkFree, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at the following Docs which will help with the suggested changes the maintainers have proposed:

Quickstart: https://linkfree.io/docs/quickstart

Editing your Profile: https://linkfree.io/docs/how-to-guides/editing

**Pull request closed without merging**